src/main/java/com/alevel/backend/service/MailService.java

```diff
@@ -0,0 +1,43 @@
+package com.alevel.backend.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.MimeMessageHelper;
+import org.springframework.stereotype.Service;
+
+import javax.mail.MessagingException;
+import javax.mail.internet.MimeMessage;
+import java.util.Random;
+
+@Service
+public class MailService {
+
+    @Autowired
+    private JavaMailSender mailSender;
+
+    public int createKey() {
+        Random random = new Random();
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

To make your code secure, use `new SecureRandom()` to instantiate a `java.security.SecureRandom` object. Do not use `getInstance()`, `getInstanceStrong()` and `java.util.Random`.